From a9723e1015e918f9800a6cf2b18ccc0a03531a26 Mon Sep 17 00:00:00 2001
From: Philip Chimento <philip.chimento@gmail.com>
Date: Tue, 30 Sep 2014 23:39:03 -0700
Subject: [PATCH 1/2] build: Don't error out on missing cairo-xlib.pc

When building on OSX, we would be using Cairo without XLib support, so
instead of erroring out if pkg-config can't find cairo-xlib, print a
warning. The X-based code will not be used anyway if Cairo hasn't
detected XLib.
---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6efd58d..aa38663 100644
--- a/configure.ac
+++ b/configure.ac
@@ -98,7 +98,8 @@ AM_CONDITIONAL(ENABLE_CAIRO, test x$have_cairo = xyes)
 AS_IF([test x$have_cairo = xyes], [
   AC_DEFINE([ENABLE_CAIRO],[1],[Define if you want to build with cairo support])
 ])
-PKG_CHECK_MODULES([GJS_CAIRO_XLIB], [cairo-xlib], , )
+PKG_CHECK_MODULES([GJS_CAIRO_XLIB], [cairo-xlib], [],
+  [AC_MSG_WARN([Cairo-xlib support not found])])
 
 # Optional GTK+ dep (enabled by default)
 AC_ARG_WITH(gtk,
-- 
1.9.3 (Apple Git-50)

